qds_sdk/commands.py

```diff
@@ -254,6 +254,98 @@ def parse(cls, args):
         v["command_type"] = "HiveCommand"
         return v
 
+class SparkCommand(Command):
+	
+	#{"program":prog,"language":"python","command_type":"SparkCommand","label":"praveen-spark"} 
+    usage = ("sparkcmd <submit|run> [options]")
+    allowedlandlist = ["python", "scala"]
+    optparser = GentleOptionParser(usage=usage)
+    optparser.add_option("--program", dest="program", help="program code")
```

Comment:
--script-location would be a location on s3 and not local box. For now we
can follow whatever other commands support via SDK.

On Sun, Jan 18, 2015 at 9:47 AM, Karandeep Johar <notifications@github.com>
wrote:

> In qds_sdk/commands.py
> <https://github.com/qubole/qds-sdk-py/pull/78#discussion-diff-23130757>:
>
> > @@ -254,6 +254,98 @@ def parse(cls, args):
> >          v["command_type"] = "HiveCommand"
> >          return v
> >
> > +class SparkCommand(Command):
> > +	
> > +	#{"program":prog,"language":"python","command_type":"SparkCommand","label":"praveen-spark"}
> > +    usage = ("sparkcmd <submit|run> [options]")
> > +    allowedlandlist = ["python", "scala"]
> > +    optparser = GentleOptionParser(usage=usage)
> > +    optparser.add_option("--program", dest="program", help="program code")
>
> You can attach a program file using --script-location argument I think.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/qubole/qds-sdk-py/pull/78/files#r23130757>.
>